cmd/dhcpserver/list.go

```diff
@@ -49,7 +49,15 @@ var listCmd = &cobra.Command{
 		}
 
 		table := utils.NewTable()
-		table.Render(dhcpservers, nil)
+		table.SetHeader([]string{"ID", "Network ID", "Network Name", "Status"})
```

Comment:
It will be good to report the leases for that network as well.

Reply:
Network leases are displayed as a part of pvsadm dhcpserver get command, Do we need to include here as well.

Reply:
okay, that's fine..